lib/remote/configstageshandler.hpp

```diff
@@ -24,6 +24,8 @@ class ConfigStagesHandler final : public HttpHandler
 		HttpServerConnection& server
 	) override;
 
+	static void ReSetRunningPackageUpdates();
```

Comment:
Feels strange to have a function only for this operation and not the others. I'd go for no functions since these are rather simple single statements anyways. (But if you want to keep the functions, I think the capitalization looks strange and it should just be "Reset".)

Reply:
Just assumed that the other classes shouldn't have the ability to set this member variable to `true` but only reset it.

Reply:
Ok, things are split across classes, so that might be a fair point. I haven't looked too much at the context so far to be honest.